build.sbt

```diff
@@ -151,6 +151,24 @@ lazy val repl = project
 //    fork in (Test, testOnly) := true
   )
 
+
+/**
+ * REPL available via remote ssh access. Pluggable into any app environment to allow live hacking on a live application.
+ */
+lazy val sshd = project.
+    dependsOn(repl).
+    settings(sharedSettings:_*).
+    settings(
+      name := "ammonite-repl-sshd",
```

Comment:
Let's just name this `ammonite-sshd`, and the package `ammonite.sshd`, rather than putting it into the `ammonite.repl` namespace